CONTRIBUTING.md

```diff
@@ -17,9 +17,12 @@ Tidy 5e Sheets builds a rich UI with [svelte](https://svelte.dev/) and leverages
 
 1. Download the repo
 2. Run `npm install`
-3. Run `npm run dev`
-4. Create a symlink from the newly-created `dist` folder to the Foundry `modules` folder as `tidy5e-sheet-kgar`
-5. Start Foundry VTT
+3. Run `npm run prepare-dev`
+4. Copy `foundry-data-path-config_example.json` to `foundry-data-path-config.json`
+5. Edit `foundry-data-path-config.json` and change `dataPath` to point to your Foundry VTT data folder
```

Comment:
After doing a run-through of this, I'd recommend a hint about what directories should be found in this data folder, because there's also a Data folder in the UserData folder, creating potential for confusion:
![image](https://github.com/kgar/foundry-vtt-tidy-5e-sheets/assets/4632129/12fff274-910e-41be-b95b-57f3206e88ec)

Maybe something like:
The `dataPath` directory should contain folders like `Backups`, `Config`, `Data`, and `Logs`.

What do you think?

Reply:
Sure thing.

Reply:
Take a look now. :)